### PR TITLE
Homepage 

### PR DESCRIPTION
**Pull request by manujjain**
Review

### CODE REVIEW

File: accounts/templates/home.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<style type="text/css">
```
Comment (HunterXHunterT9O) [inline accounts/templates/home.html]: Styling should be done later
We should focus on functionality for now
Refrain from styling in future PRs

File: accounts/templates/home.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+		.container {
+    padding: 16px;
+}
+	</style>
+</head>
+<body>
+
+<div class="container">
+<form>
+<input type="button" class="button button1" value="Create Team" onclick="window.location.href=' http://127.0.0.1:8000/create-team '" />
```
Comment (HunterXHunterT9O) [inline accounts/templates/home.html]: Relative URLs should be used instead

File: accounts/templates/login.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+       width: 100%;
+    }
+}
+</style>
+</head>
+<body>
+
+<h2>Login Form</h2>
+
+<form action="/action_page.php">
```
Comment (HunterXHunterT9O) [inline accounts/templates/login.html]: What is php for?

File: accounts/templates/login.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+  </div>
+
+  <div class="container">
+    <label for="uname"><b>Username</b></label>
+    <input type="text" placeholder="Enter Username" name="uname" required>
+
+    <label for="psw"><b>Password</b></label>
+    <input type="password" placeholder="Enter Password" name="psw" required>
+        
+    <button type="submit">Login</button>
```
Comment (HunterXHunterT9O) [inline accounts/templates/login.html]: We don't use regular form submission
We use Ajax instead
`type="submit"` should be removed

File: accounts/templates/text [hunk 5]
```diff
@@ -0,0 +1 @@
+task manager app
```
Comment (HunterXHunterT9O) [inline accounts/templates/text]: What is this file for?

File: task_manager/urls.py [hunk 6]
```diff
@@ -15,10 +15,15 @@
 """
 from django.contrib import admin
 from django.urls import path
```
Comment (HunterXHunterT9O) [inline task_manager/urls.py]: We use django 1.11.2, not 2.0
This line should be removed

File: task_manager/urls.py [hunk 7]
```diff
@@ -15,10 +15,15 @@
...
+
 
 
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    url(r'^',include('accounts.urls')),
+
     
-]
```
Comment (HunterXHunterT9O) [inline task_manager/urls.py]: A list must having a losing bracket, otherwise it'd throw an error

File: task_manager/urls.py (hunk 6 above)
Comment (manujjain) [reply task_manager/urls.py]: I just created it to create templates folder as a folder can't be empty

File: accounts/templates/text (hunk 5 above)
Comment (manujjain) [reply accounts/templates/text]: I just created it to create template folder as a folder can't be empty

File: task_manager/urls.py (hunk 6 above)
Comment (manujjain) [reply task_manager/urls.py]: what should I use instead

**Pull request merged**